examples/stream.py

```diff
@@ -0,0 +1,18 @@
+import asyncio
+
+from luno_python.stream_client import stream_market
+from luno_python.api_types import format_state
+
+def handle_update(pair, state, update):
+    print(format_state(pair, state))
+    if update is not None:
+        print(update)
+
+asyncio.get_event_loop().run_until_complete(
+    stream_market(
+        pair="XBTZAR",
+        api_key_id=[api key],
```

Comment:
Suggest using a dummy string literal so this at least runs.

Reply:
Add empty string